app/models/base_organisation.rb

```diff
@@ -54,5 +57,32 @@ def setup_categories
   def has_relation?(category)
     self.category_organisations.any? {|v| v.category_id == category.id}
   end
+  
+  def slug_candidates
+    [
+      :short_name,
+      :prolonged_name,
+      :orged,
+      :name
+    ]
+  end
```

Comment:
yes, but doesn't mean it's great from a Ruby POV.  I mean hound is complaining about long method, and it *is* a constant, so like

```ruby
SLUG_CANDIDATES = %i{short_name, prolonged_name, orged, name}
```

would be more Ruby-ish I think ... ah, but maybe the gem is relying on the method called `slug_candidates` to exist, which maybe means that it has to be there in which case I might do:

```ruby
def slug_candidates
  SLUG_CANDIDATES
end
```

effectively extracting the constant ... maybe that's premature, but I think it's good to flag to other developers that this is something that is constant, by making it an explicit constant ...